Create email-templates-examples.md (#1228)
bigcommerce/dev-docs: docs/api-docs/management-apis/email-templates/email-template-examples.md

# Email Templates Code Examples

<div class="otp" id="no-index">

### On this page

- [Updating text](#updating-text)
- [Adding a button](#adding-a-button)
- [Changing logo size](#changing-logo-size)
- [Changing font size](#changing-font-size)

</div>

> This article provides examples of how to customize email templates. The examples are for use in the recently updated templates. You can enable the new version of transactional emails by going to **Marketing> Transactional Emails** and clicking **Try New Experience**. When you are ready to use a template, make sure you **Enable** the template from the respective dropdown. 

Reviewer: ```suggestion
This article provides examples of how to customize email templates. The examples are for use in the recently updated templates. You can enable the new version of transactional emails by going to **Marketing > Transactional Emails** and clicking **Try New Experience**. When you are ready to use a template, make sure to **Enable** the template from the respective dropdown. 
```

> Code examples below will demonstrate how to update text, add a button, change logo size, and change the font size.

Reviewer: ```suggestion
The following code examples demonstrate how to update text, add a button, change logo size, and change the font size.
```

<div class="HubBlock--callout">
<div class="CalloutBlock--info">
<div class="HubBlock-content">
  
  ### Note 

>   It is important to preview all changes and style your emails for accessibility.  

Reviewer: I feel like this note doesn't provide me with sufficient information to take action. Perhaps you can link to the [Developing Accessible Themes](https://developer.bigcommerce.com/stencil-docs/accessibility/theme-accessibility) article to give readers more context.

  
</div>
</div>
</div> 

## Updating text

> To change existing text in an email template, you will need to update information in the **Phrases** and **Code** tabs. For detailed instructions, see [Editing Template Phrases](https://support.bigcommerce.com/s/article/Customizing-Emails?language=en_US#phrases). 

Reviewer: We should try to avoid future tense and use present tense instead. 
```suggestion
To change existing text in an email template, update information in the **Phrases** and **Code** tabs. For detailed instructions, see [Editing Template Phrases](https://support.bigcommerce.com/s/article/Customizing-Emails?language=en_US#phrases). 
```

## Adding a button

> To add a button to an email, you will need to add additional code to your template. Here is example HTML code that will add a blue button with white text to the account created template.

Reviewer: ```suggestion
You can create a responsive email button using HTML and CSS. The following code example adds a blue button with white text to the account created template.
```
I tried to make it more straightforward. Please ignore my suggestion if it changes the meaning.

> Go to **Transactional Emails> Email Templates**. Click **...** next to **Account Created** and select **Edit Template**. Copy and paste the contents below into the Content editor. Ensure you paste the text outside the `<table>` tags. 

Reviewer: ```suggestion
Go to **Transactional Emails > Email Templates**. Click **...** next to **Account Created** and select **Edit Template**. Copy and paste the contents below into the Content editor. Ensure you paste the text outside the `<table>` tags. 
```
This part is confusing to me - "Ensure you paste the text outside the `<table>` tags." What does that mean?

  
```html
<table class="row">
  <tr>
    <th class="column">
       <table>
          <tr>
             <th>
                <a href="{{store.path}}" style = "background-color:blue; color:white" class="sigh-in">{{lang 'sign_in'}}</a>
             </th>
             <th class="expander"></th>
          </tr>
      </table>
  </th>
 </tr>
</table>
```

<div class="HubBlock--callout">
<div class="CalloutBlock--info">
<div class="HubBlock-content">
  
  ### Note 

>   You can select a different button background and text color. We suggest adhering to the minimum color contrast standards set by [WCAG](https://webaim.org/articles/contrast/). 

Reviewer: ```suggestion
  You can select a different button background and text color. We suggest adhering to the [minimum color contrast standards](https://webaim.org/articles/contrast/) set by WCAG. 
```

</div>
</div>
</div> 

> ![Add button](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/email-templates-add-button.png "Button with white text on a blue background")

Reviewer: The photo didn't display for me.

## Changing logo size

> To change the size of your logo that appears in your email, specify width and height values in the Content editor. Go to **Transactional Emails> Email Templates**. Click **...** next to **Order Email** and select **Edit Template**. Go to the **Code** tab and replace: 

Reviewer: ```suggestion
To change the size of your logo that appears in your email, specify width and height values in the Content editor. Go to **Transactional Emails > Email Templates**. Click **...** next to **Order Email** and select **Edit Template**. Go to the **Code** tab and replace: 
```

```html
<img src="{{store.logo.url}}" alt="{{store.logo.title}}">
```
with
```html
<img src="{{store.logo.url}}" alt="{{store.logo.title}}" width="200" height="200">
```
<div class="HubBlock--callout">
<div class="CalloutBlock--info">
<div class="HubBlock-content">
  
  ### Note  
  You can also add width and height values to images.
</div>
</div>
</div> 
  

> ![Change logo size](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/email-templates-change-logo-size.png "Change logo size")

Reviewer: The photo didn't display for me.

## Changing font size
  

> To change the email font size, you will need to use the style attribute. You can define attributes such as color and font size within the paragraph `<p>` tag. Go to **Transactional Emails> Email Templates**, click **...** next to **Guest Account**, and select **Edit Template**. Go to the **Code** tab and replace: 

Reviewer: ```suggestion
To change the email font size, use the style attribute. You can define attributes such as color and font size within the paragraph `<p>` tag. Go to **Transactional Emails > Email Templates**, click **...** next to **Guest Account**, and select **Edit Template**. Go to the **Code** tab and replace: 
```

  
```html
<p>{{lang 'help'}}</p>
  ```
 with 
  
  ```html
<p style="font-size:16px;">{{lang 'help'}}</p>
   ```

> To make your text responsive so that it can viewed on a variety of devices and screen sizes, you can change the text to the following:

Reviewer: ```suggestion
To make your text responsive so that it can viewed on a variety of devices and screen sizes, use viewport width (`vw`) unit to set the `font-size`.
```

  
  ```html
  <p style="font-size:2.0vw">{{lang 'help'}}</p>
  ```
<div class="HubBlock--callout">
<div class="CalloutBlock--info">
<div class="HubBlock-content">
  
  ### Note 

>   The `vw` style settings allow you to control the widths of elements no matter the viewport size.  

Reviewer: This note can be incorporated in the text above.

# Email Templates Code Examples

<div class="otp" id="no-index">

### On this page

- [Updating text](#updating-text)
- [Adding a button](#adding-a-button)
- [Changing logo size](#changing-logo-size)
- [Changing font size](#changing-font-size)

</div>

This article provides examples of how to customize email templates. The examples are for use in the the recently updated templates. You can enable the new version of transactional emails by going to **Marketing> Transactional Emails** and clicking **Try New Experience**. When you are ready to use a template, make sure you **Enable** the template from the respective dropdown. 

Code examples below will demonstrate how to update text, add a button, change logo size, and change the font size.

<div class="HubBlock--callout">
<div class="CalloutBlock--info">
<div class="HubBlock-content">

> NOTE: It is important to preview all changes and style your emails to be accessible for everyone.  

Reviewer 2: ```suggestion
#### Note
> It is important to preview all changes and style your emails to be accessible for everyone.  
```

</div>
</div>
</div> 

> ## Updating text

Reviewer 2: Is the information in this section replicated in the KB? 

If so, I recommend linking to it instead of recreating it here since any change to the instructions will need to be updated in two separate places by two separate teams down the line.

  <tr>
    <td>
      <table class="row">
        <tr>
          <th class="column">
            <table>
              <tr>
                <th>
                  <table class="delimiter">
                    <tr>
                     <th>
                       <hr>
                </th>
                     <th class="delimiter__image-block">
                        <img src="{{store.cdn_path}}/img/emails/shop.png" alt="Shop image" class="delimiter__image">
                     </th>
                     <p>{{lang 'security'}}</p>
                     <th>
                      <hr>
                     </th>
                   </tr>
                 </table>
                </th>
              </tr>
              <th class="expander"></th>
          </tr>
         </table>
        </td>
   </tr>
 </table>
```
Click on the **Preview** tab to view your changes.
[Image]()

## Adding a button

To add a button to an email, you will need to add additional code to your template. Here is an example of HTML code that will add a blue button with white text to the account created template.

<div class="HubBlock--callout">
<div class="CalloutBlock--info">
<div class="HubBlock-content">

> NOTE: You can select a different button background and text color. We suggest adhering to the minimum color contrast standards set by [WCAG](https://webaim.org/articles/contrast/). 

Reviewer 2: ```suggestion
#### Note 
You can select a different button background and text color. We suggest adhering to the minimum color contrast standards set by [WCAG](https://webaim.org/articles/contrast/). 
```

Reviewer 2: Mone this note to the bottom of the section

```html
<img src="{{store.logo.url}}" alt="{{store.logo.title}}">
```
with
```html
<img src="{{store.logo.url}}" alt="{{store.logo.title}}" width="200" height="200">
```
<div class="HubBlock--callout">
<div class="CalloutBlock--info">
<div class="HubBlock-content">
NOTE:  You can also add width and height values to images.
</div>
</div>
</div> 
  
[Image]

## Changing font size
  
To change the your email font size, you will need to use the style attribute. You can define attributes such as color and font size within the paragraph `<p>` tag. Go to **Transactional Emails> Email Templates**, click **...** next to **Guest Account**, and select **Edit Template**. Go to the **Code** tab and replace: 
  
```html
<p>{{lang 'help'}}</p>
  ```
 with 
  
  ```html
<p style="font-size:16px;">{{lang 'help'}}</p>
   ```
To make your text responsive so that it can viewed on a variety of devices and screen sizes, you can change the text to the following:
  
  ```html
  <p style="font-size:2.0vw">{{lang 'help'}}</p>
  ```
<div class="HubBlock--callout">
<div class="CalloutBlock--info">
<div class="HubBlock-content">

> NOTE: vw style settings allow you to control the widths of elements no matter the viewport size.  

Reviewer 2: ```suggestion
#### Note 
The `vw` style settings allow you to control the widths of elements no matter the viewport size.  
```

</div>

This article provides examples of how to customize email templates. The examples are for use in the recently updated templates. You can enable the new version of transactional emails by going to **Marketing> Transactional Emails** and clicking **Try New Experience**. When you are ready to use a template, make sure you **Enable** the template from the respective dropdown. 

Code examples below will demonstrate how to update text, add a button, change logo size, and change the font size.

<div class="HubBlock--callout">
<div class="CalloutBlock--info">
<div class="HubBlock-content">
  
  ### Note 
  It is important to preview all changes and style your emails for accessibility.  
  
</div>
</div>
</div> 

## Updating text
To change existing text in an email template, you will need to update information in the **Phrases** and **Code** tabs. For detailed instructions, see [Editing Template Phrases](https://support.bigcommerce.com/s/article/Customizing-Emails?language=en_US#phrases). 

## Adding a button

> To add a button to an email, you will need to add additional code to your template. Here is example HTML code that will add a blue button with white text to the account created template.

Author: To add a button to an email, you will need to add additional code to your template. Below is example HTML code that adds a blue button with white text to the account created template.
